sklearn/metrics/cluster/tests/test_common.py

```diff
@@ -161,7 +161,9 @@ def generate_formats(y):
         y = np.array(y)
         yield y, 'array of ints'
         yield y.tolist(), 'list of ints'
-        yield [str(x) for x in y.tolist()], 'list of strs'
+        yield [str(x) + "-a" for x in y.tolist()], 'list of strs'
```

Comment:
Ooh... right. You need to make it a non-numeric string.

Reply:
yes, kind of. Though in master string dtypes are just passed through (with a deprecation warning).

Just adding the first line still passed, though wouldn't pass any more after we complete the deprecation (when we would try to make it a float).